src/types/shared.ts

```diff
@@ -1,3 +1,17 @@
 import { Type } from "@sinclair/typebox";
 
 export const TURL = Type.String({ format: "ipv4" });
+
+export interface StreamlinedComment {
+  login: string | null | undefined;
```

Comment:
Would you follow up other schema/type definitions for StreamlinedComment and GPTResponse.

Reply:
no worries and cheers